cmd/packageImport.go

```diff
@@ -86,20 +93,20 @@ func newPackageImportCommand() *packageImportCommand {
 	cmd.Flags().StringSliceVarP(&remoteRepos, flagRepoStructure, "r", make([]string, 0), "create an importable config based on on the structure of a remote repository")
 	cmd.Flags().StringSliceVarP(&directories, flagDirectoryStructure, "d", make([]string, 0), "create an importable config based on the structure of a local directory")
 	cmd.Flags().StringP(flagExclude, "e", "", "regex pattern to exclude paths from import (only works with -c, -r, -d)")
+	cmd.Flags().BoolVarP(&fromStdin, flagStdin, "i", false, "import a configuration from standard input")
 
 	_ = cmd.MarkFlagDirname(flagDirectoryStructure)
 	_ = cmd.MarkFlagFilename(flagConfig)
 
 	return &packageImportCommand{cmd: cmd}
 }
 
-func importPackage(status *statuswriter.Sink, path, importType string, exclude *regexp.Regexp) {
+func importPackageWithPath(status *statuswriter.Sink, path, importType string, exclude *regexp.Regexp) {
```

Comment:
This function should be renamed back to `importPackage` again because its kinda a meta-importer function importing stuff from everywhere.
In order to support `stdin-imports` we need another case in the switch statement below. Something like this should do: 

```go
case flagStdin:
    pkg, err = importPackageFromStdin(status)
```

As you can see the signature of the `importPackageFromStdin` function would change, making it comply with the other import functions.